packages/block-editor/src/components/block-alignment-matrix-toolbar/style.scss

```diff
@@ -0,0 +1,7 @@
+.block-editor-block-alignment-matrix-toolbar {
+	.components-popover__content {
+		min-width: 0;
+		padding: $grid-unit;
+		width: auto;
+	}
```

Comment:
Curious to know why we need custom styling and whether these could be abstracted somehow in Dropdown.

Reply:
Currently needed as there's no other way to style the dropdown menu.
It would be lovely to abstract it to Dropdown - perhaps out of scope for this PR?